composer.json

```diff
@@ -30,8 +30,6 @@
         "php": "8.1.*||8.2.*||8.3.*||8.4.*",
         "ext-zip": "*",
         "composer/composer": "^2.8.2",
-        "guzzlehttp/guzzle": "^7.8",
-        "guzzlehttp/psr7": "^2.6",
         "illuminate/container": "^10.47",
         "psr/http-message": "^2.0",
```

Comment:
Shouldn't `psr/http-message` also be removed ? It is used by guzzle by I cannot find other usages

Reply:
Quite possibly; I will check the deps again now :)